packages/suite/src/views/wallet/send/Outputs/TokenSelect/tokenSelectUtils.ts

```diff
@@ -0,0 +1,118 @@
+
+export const buildTokenOptions = (
+    accountTokens: Account['tokens'],
+    symbol: Account['symbol'],
+    coinDefinitions: TokenDefinitions['coin'],
+    nativeTokenBalance: Account['formattedBalance'],
+    translationString: TranslationFunction,
+) => {
+    // native token option
+    const result: SelectAssetOptionCurrencyProps[] = [
+        {
+            type: 'currency',
+            symbol,
+            networkSymbol: symbol,
+            coingeckoId: networks[symbol].coingeckoNativeId || '',
+            contractAddress: null,
+            cryptoName: networks[symbol].name,
+            balance: nativeTokenBalance,
+        },
+    ];
+
+    if (accountTokens) {
+        const tokens = getTokens(accountTokens, symbol, coinDefinitions);
+
+        tokens.shownWithBalance.forEach(token => {
+            result.push({
+                type: 'currency',
+                symbol: token.symbol ?? symbol,
+                networkSymbol: symbol,
+                coingeckoId: getCoingeckoId(symbol) ?? '',
+                contractAddress: token.contract,
+                cryptoName: token.name,
+                balance: token.balance,
+                tokenExplorerUrl: getTokenExplorerUrl(networks[symbol], token),
+            });
+        });
+
+        if (tokens.hiddenWithBalance.length) {
+            tokens.hiddenWithBalance.forEach(token => {
+                result.push({
+                    type: 'currency',
+                    symbol: token.symbol ?? symbol,
+                    networkSymbol: symbol,
+                    hidden: true,
+                    coingeckoId: getCoingeckoId(symbol) ?? '',
+                    contractAddress: token.contract,
+                    cryptoName: token.name,
+                    balance: token.balance,
+                });
+            });
+        }
+
+        if (tokens.unverifiedWithBalance.length) {
+            tokens.unverifiedWithBalance.forEach(token => {
+                result.push({
+                    type: 'currency',
+                    unverified: true,
+                    symbol: token.symbol ?? symbol,
+                    networkSymbol: symbol,
+                    coingeckoId: getCoingeckoId(symbol) ?? '',
+                    contractAddress: token.contract,
+                    cryptoName: token.name,
+                    balance: token.balance,
+                    badge: translationString('TR_UNRECOGNIZED'),
+                });
+            });
+        }
+    }
+
+    return result;
+};
+
+export const sendTokenCategories = (
+    translationString: TranslationFunction,
+): TokenFilterCategories => ({
+    categoriesType: 'tokens',
+    categories: [
+        {
+            type: 'visibleWithBalance',
+            label: translationString('TR_TOKENS'),
+        },
+        {
+            type: 'hiddenWithBalance',
+            label: translationString('TR_HIDDEN'),
+        },
+    ],
+});
+
+export const copyAddressAction =
```

Comment:
move this to another file as it is reused from two places and it is action, not util